### PR TITLE
DependBot : Upgrade libs versions (Rdcc 2507)

### PR DESCRIPTION
**Pull request by pksaxena01**
### JIRA link (if applicable) ###
RDCC-2507

### Change description ###
This PR addressed the Jar bump up versions and audit ison issues

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -29,13 +29,6 @@
     <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed/tomcat\-embed\-websocket@.*$</packageUrl>
     <cve>CVE-2020-13943</cve>
   </suppress>
-  <suppress>
-    <notes><![CDATA[
-   file name: launchdarkly-java-server-sdk-5.1.1.jar (shaded: com.google.guava:guava:28.2-jre)
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/com\.google\.guava/guava@.*$</packageUrl>
-    <cve>CVE-2020-8908</cve>
-  </suppress>
```
Comment (shreedhar-hmcts) [inline config/owasp/suppressions.xml:38]: CVE removed, hoping also removed from confluence page list

**Review by shreedhar-hmcts — APPROVED** [review]

**Review by SujeethKumarAri — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -331,7 +331,7 @@ dependencies {
 
   compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.11.2'
   compile group: 'io.jsonwebtoken', name: 'jjwt', version:'0.9.1'
-  compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.11.2'
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.12.1'
```
Comment (abhijitdiwan) [inline build.gradle:334]: can't we use 2.11.2 same?

File: build.gradle [hunk 3]
```diff
@@ -449,14 +449,14 @@ dependencies {
     exclude group: "org.hamcrest", module: "hamcrest-library"
   }
 
-    testCompile group: 'io.rest-assured', name: 'rest-assured', version: '4.0.0'
-    testCompile group: 'io.rest-assured', name: 'rest-assured-common', version: '4.2.0'
+    testCompile group: 'io.rest-assured', name: 'rest-assured', version: '4.3.3'
+    testCompile group: 'io.rest-assured', name: 'rest-assured-common', version: '4.3.3'
```
Comment (abhijitdiwan) [inline build.gradle:453]: same version in constants hence will update both io.rest-assured in future.

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (pksaxena01) [reply config/owasp/suppressions.xml:38]: Removed entry for CRD in Confluence page.However there are other repos too so still you can find this CVE for others

File: build.gradle (hunk 2 above)
Comment (pksaxena01) [reply build.gradle]: Done

File: build.gradle (hunk 3 above)
Comment (pksaxena01) [reply build.gradle]: Done

**Review by SujeethKumarAri — DISMISSED** [review]

**Review by SujeethKumarAri — APPROVED** [review]

**Review by abhijitdiwan — APPROVED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (shreedhar-hmcts) [reply config/owasp/suppressions.xml:38]: ok please merge then

**Pull request merged**